src/main/scala/no/ndla/conceptapi/controller/DraftConceptController.scala

```diff
@@ -10,59 +10,40 @@ package no.ndla.conceptapi.controller
 import no.ndla.conceptapi.auth.User
-import no.ndla.conceptapi.model.api.{
-  Concept,
-  ConceptSearchParams,
-  ConceptSearchResult,
-  Error,
-  NewConcept,
-  NotFoundException,
-  SubjectTags,
-  TagsSearchResult,
-  UpdatedConcept,
-  ValidationError
-}
-import no.ndla.conceptapi.model.domain.{Language, SearchResult, Sort}
+import no.ndla.conceptapi.model.api.{Concept, ConceptSearchParams, ConceptSearchResult, NotFoundException}
+import no.ndla.conceptapi.model.domain.{ConceptStatus, Language, SearchResult, Sort}
 import no.ndla.conceptapi.model.search.SearchSettings
-import no.ndla.conceptapi.service.search.{ConceptSearchService, SearchConverterService}
-import no.ndla.conceptapi.service.{ReadService, WriteService}
+import no.ndla.conceptapi.service.search.{DraftConceptSearchService, SearchConverterService}
+import no.ndla.conceptapi.service.{ConverterService, ReadService, WriteService}
 import org.json4s.{DefaultFormats, Formats}
-import org.scalatra.swagger.{ResponseMessage, Swagger, SwaggerSupport}
-import org.scalatra.{Created, NotFound, Ok}
+import org.scalatra.Ok
+import org.scalatra.swagger.{Swagger, SwaggerSupport}
 
 import scala.util.{Failure, Success}
 
-trait ConceptController {
-  this: WriteService with ReadService with User with ConceptSearchService with SearchConverterService =>
-  val conceptController: ConceptController
-
-  class ConceptController(implicit val swagger: Swagger) extends NdlaController with SwaggerSupport with LazyLogging {
+trait DraftConceptController {
+  this: WriteService
+    with ReadService
+    with User
+    with DraftConceptSearchService
+    with SearchConverterService
+    with DraftNdlaController
+    with ConverterService =>
+  val draftConceptController: DraftConceptController
+
+  class DraftConceptController(implicit val swagger: Swagger)
+      extends DraftNdlaControllerClass
+      with SwaggerSupport
+      with LazyLogging {
     protected implicit override val jsonFormats: Formats = DefaultFormats
-    private val conceptId =
-      Param[Long]("concept_id", "Id of the concept that is to be returned")
-
     val applicationDescription = "This is the Api for concepts"
 
-    // Additional models used in error responses
-    registerModel[ValidationError]()
-    registerModel[Error]()
-
-    val response400 =
-      ResponseMessage(400, "Validation Error", Some("ValidationError"))
-    val response403 = ResponseMessage(403, "Access Denied", Some("Error"))
-    val response404 = ResponseMessage(404, "Not found", Some("Error"))
-    val response500 = ResponseMessage(500, "Unknown error", Some("Error"))
-
-    private val query =
-      Param[Option[String]]("query", "Return only concepts with content matching the specified query.")
-    private val conceptIds = Param[Option[Seq[Long]]](
-      "ids",
-      "Return only concepts that have one of the provided ids. To provide multiple ids, separate by comma (,).")
+    private val statuss = Param[String]("STATUS", "Concept status")
```

Comment:
Kvifor ha to ss-er? Er status et beskytta ord i scala?

Reply:
Ikke i scala, men det er en variabel i scalatra som er tilgjengelig i ScalatraServlets.
Hadde fint kunne hett `status` men da ville den blitt shadowa.